### PR TITLE
Kill default content type

### PR DESCRIPTION
**Pull request by greypants**
This was preventing Superagent from intelligently determining the correct content-type header to send. Since `application/json` is already the default type, there's no need to enforce it here.

Also adds a `.nvmrc` file. Tests fail on Node 6. Set it to `0.12.18`

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v0.12.18
```
Comment (nhunzaker) [inline .nvmrc]: What fails on 6.0?

**Comment by nhunzaker** [comment]
👍

File: .nvmrc (hunk 1 above)
Comment (greypants) [inline .nvmrc]:
```bash
●° gangway git:(kill-default-content-type) nvm use 6
Now using node v6.9.2 (npm v3.10.9)
●° gangway git:(kill-default-content-type) npmm run test
zsh: correct 'npmm' to 'npm' [nyae]? n
zsh: command not found: npmm
●° gangway git:(kill-default-content-type) npm run test

> gangway@3.0.0-rc test /Users/greypants/Sites/gangway
> mocha -R dot



  ․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․

  59 passing (570ms)
  15 failing

  1) ajax can send requests:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  2) ajax can handle errors:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  3) ajax can operate as a promise a response:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  4) ajax can convert bindings using params:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  5) ajax falls back on the `body` param if no params are given:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  6) ajax can be converted into a node callback:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  7) ajax can parse a response:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  8) ajax can parse an error:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  9) ajax Promise decoration can chain off of requests:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  10) ajax Promise decoration can catch rejected promises:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  11) ajax Promise decoration supports the done() Promise method:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  12) ajax Promise decoration when done() raises an error done errors bubble out:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  13) ajax Promise decoration when done() raises an error done errors bubble out if given no callback:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  14) API() when a route is executed folds together configuration:
     Uncaught TypeError: handle.writeLatin1String is not a function
      at createWriteReq (net.js:738:21)
      at Socket._writeGeneric (net.js:708:11)
      at Socket._write (net.js:731:8)
      at doWrite (_stream_writable.js:334:12)
      at clearBuffer (_stream_writable.js:441:7)
      at Socket.Writable.uncork (_stream_writable.js:270:7)
      at ClientRequest._flushOutput (_http_outgoing.js:678:10)
      at ClientRequest.OutgoingMessage._flush (_http_outgoing.js:652:16)
      at _http_client.js:197:10
      at callSocketMethod (_http_client.js:597:7)
      at ClientRequest.onSocket (_http_client.js:602:7)
      at tickOnSocket (_http_client.js:568:7)
      at onSocketNT (_http_client.js:580:5)
      at _combinedTickCallback (internal/process/next_tick.js:74:11)
      at process._tickCallback (internal/process/next_tick.js:98:9)

  15) API() when a namespaced route is executed "before each" hook for "properly builds URLs for namespaces":
     Error: faux-jax: Cannot call `install()` twice. Did you forgot to call `restore()`?
      at FauxJax.install (node_modules/faux-jax/index.js:20:24)
      at Context.<anonymous> (test/api.test.js:167:15)




npm ERR! Darwin 16.4.0
npm ERR! argv "/Users/greypants/.nvm/versions/node/v6.9.2/bin/node" "/Users/greypants/.nvm/versions/node/v6.9.2/bin/npm" "run" "test"
npm ERR! node v6.9.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! gangway@3.0.0-rc test: `mocha -R dot`
npm ERR! Exit status 15
npm ERR!
npm ERR! Failed at the gangway@3.0.0-rc test script 'mocha -R dot'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the gangway package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     mocha -R dot
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs gangway
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls gangway
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/greypants/Sites/gangway/npm-debug.log
```

Comment (nhunzaker) [inline .nvmrc]: Jeez. Okay. I wonder if Mocha is just super out of date or something.

**Pull request merged**